src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const lines = sourceString.split(';');
+  const result = {};
+
+  lines.forEach((line) => {
+    const trimmedLine = line.trim();
+
+    if (trimmedLine === '' || !trimmedLine.includes(':')) {
+      return;
+    }
 
+    const [property, ...valueParts] = trimmedLine.split(':');
```

Comment:
Using spread operator to capture valueParts is not necessary here. Since you are only interested in the first split for the property and the rest as the value, consider using destructuring with two variables, like this: 'const [property, value] = trimmedLine.split(':').map(part => part.trim());'. This also eliminates the need for the join operation on line 20.